antlr/src/main/antlr4/org/apache/iotdb/db/qp/strategy/SqlBase.g4

```diff
@@ -19,6 +19,8 @@
 
 grammar SqlBase;
 
+@parser::members {public static boolean isID;}
```

Comment:
where to set the `isID=true`?

Reply:
I'll change it to make it more understandable.